src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,20 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+        <UserInfo user={post.user} />
```

Comment:
find can returns undefined, so it's better to add some check here
```suggestion
        {post.user && <UserInfo user={post.user} />}
```